From f21230c04eb9a5e0152f670695bf1a9bd9545f8c Mon Sep 17 00:00:00 2001
From: "Bernhard R. Link" <brlink@debian.org>
Date: Sun, 10 Jun 2012 11:32:08 +0200
Subject: [PATCH] debchange: fix NMU detection to not NMU QA packages

Orphaned packages can by definition not be NMUed, so fix
the NMU detection to not trigger if the maintainer contains
the packages@qa.debian.org address.
---
 scripts/debchange.pl |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/scripts/debchange.pl b/scripts/debchange.pl
index 17a4d3d..e77a632 100755
--- a/scripts/debchange.pl
+++ b/scripts/debchange.pl
@@ -862,7 +862,8 @@ if ($opt_auto_nmu eq 'yes' and ! $opt_v and ! $opt_l and ! $opt_s and
 
 	    my $packager = "$MAINTAINER <$EMAIL>";
 
-	    if (! grep { $_ eq $packager } ($maintainer, @uploaders) and
+	    if ($maintainer !~ m/<packages\@qa.debian.org>/ and
+                ! grep { $_ eq $packager } ($maintainer, @uploaders) and
 		$packager ne $changelog{'Maintainer'} and ! $opt_team) {
 		$opt_n=1;
 		$opt_a=0;
-- 
1.7.10

